From: stephen hemminger <shemminger@vyatta.com>
Date: Thu, 5 Jan 2012 19:10:25 +0000
Subject: [PATCH 26/58] netdev: make net_device_ops const

commit e5686ad82ca2aeed7a8f24ffca115c0b7478dec9 upstream.

More drivers where net_device_ops should be const.

Signed-off-by: Stephen Hemminger <shemminger@vyatta.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
[bwh: Restrict to drivers/net/ethernet/emulex/benet/]
---
 drivers/net/ethernet/emulex/benet/be_main.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.2.46/drivers/net/ethernet/emulex/benet/be_main.c
===================================================================
--- linux-3.2.46.orig/drivers/net/ethernet/emulex/benet/be_main.c	2013-07-26 19:21:24.000000000 +0000
+++ linux-3.2.46/drivers/net/ethernet/emulex/benet/be_main.c	2013-07-26 19:21:26.000000000 +0000
@@ -3073,7 +3073,7 @@
 	return status;
 }
 
-static struct net_device_ops be_netdev_ops = {
+static const struct net_device_ops be_netdev_ops = {
 	.ndo_open		= be_open,
 	.ndo_stop		= be_close,
 	.ndo_start_xmit		= be_xmit,
